### PR TITLE
Forum Page - created connection with hashtag api

### PR DESCRIPTION
**Pull request by Bcastillo44**
- Related Issue (include '#'): #668 

- Description of changes made:

     Created connection hashtags from API to topics of interest on Forum page

- Is the feature complete/bug resolved/etc..:

     Only three hashtags at the moment, may add more today

### CODE REVIEW

File: client/src/pages/ForumPage/ForumPage.js [hunk 1]
```diff
@@ -19,8 +22,32 @@ function pillClick(val) {
...
+    .then(response => {
+      this.setState({ 
+        hashtags: response.data
+      })
+      console.log(response.data);
+    })
+    .catch(error => {
+      console.log('Error', error);
+    })
+}
```
Comment (0xZakk) [inline client/src/pages/ForumPage/ForumPage.js]: All API calls should take place in side the appropriate service in the `services/` directory. Can you move this there?

File: client/src/pages/ForumPage/ForumPage.js [hunk 2]
```diff
@@ -19,8 +22,32 @@ function pillClick(val) {
...
+    }
+  }
+
+componentDidMount() {
+  axios.get(`${BASE_URL}/hashtags`)
+    .then(response => {
+      this.setState({ 
+        hashtags: response.data
+      })
+      console.log(response.data);
```
Comment (0xZakk) [inline client/src/pages/ForumPage/ForumPage.js]: Can you remove this and any other `console.log`s?

**Review by 0xZakk — CHANGES_REQUESTED** [review]
@Bcastillo44 See my feedback. Can you also update your branch with develop?

File: client/src/pages/ForumPage/ForumPage.js (hunk 1 above)
Comment (Bcastillo44) [reply client/src/pages/ForumPage/ForumPage.js]: Added Forum.js page in services/ directory and created import/export of axios fetch for hastags on ForumPage.js

File: client/src/pages/ForumPage/ForumPage.js (hunk 2 above)
Comment (Bcastillo44) [reply client/src/pages/ForumPage/ForumPage.js]: Removed console.logs from the page

**Review by 0xZakk — CHANGES_REQUESTED** [review]
@Bcastillo44 Looks pretty good, but it looks like you've got some styling stuff to finish up:

Screen Shot 2020-07-13 at 1 45 53 PM

**Review by 0xZakk — CHANGES_REQUESTED** [review]
Can you remove the link to the sample tag?

Also, when I click to see a tag, I just get a blank tag. Is that what we expect? Could we have it link to the sample page?

**Review by 0xZakk — APPROVED** [review]

**Pull request merged**